src/main/java/com/thebois/views/PositionAdapter.java

```diff
@@ -24,4 +24,15 @@ public static Vector2 toVector(Position position) {
         return new Vector2(position.getPosX(), position.getPosY());
     }
 
+    /**
+     * Converts vector to custom position.
+     *
+     * @param vector2 The vector to convert from.
+     *
+     * @return The same vector in position form.
+     */
+    public static Position toPosition(Vector2 vector2) {
```

Comment:
Think it would be better with `vector2D`, since the current name seems to refer to the 2nd vector.

Reply:
Or just `vector` since the dimensionality of it is apparent from its type.

Reply:
Vector it is.